### PR TITLE
fix(code-examples): satisfy ESLint rules for core, forms, indicators, and inline form code examples

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2432/storybook/)

Component Storybooks:

- _(no component storybooks affected in this pr)_ 

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2432/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2432/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2432/playground/)

File: apps/code-examples/src/app/code-examples/forms/input-box/basic/demo.component.spec.ts [hunk 1]
```diff
@@ -108,12 +112,14 @@ describe('Basic input box demo', () => {
         dataSkyId: 'input-box-favorite-color',
       });
 
-      const selectEl = document.querySelector(
+      const selectEl = document.querySelector<HTMLInputElement>(
```
Comment (Blackbaud-PaulCrowder) [inline apps/code-examples/src/app/code-examples/forms/input-box/basic/demo.component.spec.ts]: Why did this change from `HTMLSelectElement` to `HTMLInputElement`?

Comment (Blackbaud-SteveBrush) [reply apps/code-examples/src/app/code-examples/forms/input-box/basic/demo.component.spec.ts]: Good catch; I was so used to typing `HTMLInputElement` for the other changes. 😅

**Review by Blackbaud-PaulCrowder — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See https://github.com/blackbaud/skyux/pull/2444

**Pull request merged**